functions/api/v1/download-latest-installer/[arch].js

```diff
@@ -0,0 +1,42 @@
+const MAVEN = "https://maven.quiltmc.org/repository/release"
+
+const UNIVERSAL_ARCH = "java-universal"
+const UNIVERSAL_PATH = MAVEN + "/org/quiltmc/quilt-installer/"
+
+const NATIVE_PATH = MAVEN + "/org/quiltmc/quilt-installer-native-bootstrap/"
+
+const METADATA = "maven-metadata.xml"
+const LATEST_REGEX = /<latest>([0-9.]+)<\/latest>/
```

Comment:
Maybe shouldn't use this and do it by version comparison - just so that if we release a patch version for an old installer or something someday it won't start giving people outdated versions.